.github/workflows/pkgci_regression_test_amdgpu_vulkan.yml

```diff
@@ -20,7 +20,7 @@ on:
 jobs:
   linux_x86_64:
     name: Linux (x86_64)
-    runs-on: nodai-amdgpu-w7900-x86-64
+    runs-on: nodai-amdgpu-mi210-x86-64
```

Comment:
FYI, if you want to test only a few specific workflows, you can skip others by putting a git trailer at the end of the PR description, like so:

`ci-exactly: build_packages, regression_test_amdgpu_rocm, regression_test_amdgpu_vulkan`

full docs (with screenshots): https://iree.dev/developers/general/contributing/#ci-behavior-manipulation

Reply:
(That's something custom we set up, doesn't work across other repos ;P)